packages/api/src/schemas/events.ts

```diff
@@ -5,6 +5,40 @@ import {
 } from "temporal-zod";
 import { z } from "zod";
 
+const conferenceEntryPointSchema = z.object({
+  entryPointType: z.enum(["video", "phone"]),
+  meetingCode: z.string(),
+  password: z.string(),
+  uri: z.string(),
+});
+
+const conferenceSchema = z.object({
+  conferenceId: z.string().optional(),
+  conferenceSolution: z
+    .object({
+      iconUri: z.string().optional(),
+      key: z
+        .object({
+          type: z.string().optional(),
+        })
+        .optional(),
+      name: z.string().optional(),
+    })
+    .optional(),
+  createRequest: z
+    .object({
+      requestId: z.string().optional(),
+      status: z.object({ statusCode: z.string().optional() }).optional(),
+      conferenceSolutionKey: z
+        .object({ type: z.string().optional() })
+        .optional(),
+    })
+    .optional(),
+  entryPoints: z.array(conferenceEntryPointSchema).optional(),
+  notes: z.string().optional(),
+  parameters: z.record(z.unknown()).optional(),
+});
```

Comment:
this is really specific on google calendar.

try using a more generic format, so we can add support for microsoft in the future as well